frontend/src/features/entity_selector/AutoEntityButton.tsx

```diff
@@ -88,31 +91,49 @@ const AutoEntityButton = ({
   }
 
   return (
-    <Tooltip title={`${t('entitySelector.newVideo')}`} aria-label="new_video">
-      {/* A non-disabled element, such as <span>, is required by the Tooltip
-          component to properly listen to fired events. */}
-      <span>
-        <Button
-          fullWidth={variant === 'full' ? true : false}
+    <>
+      {smallScreen && variant === 'compact' ? (
+        <IconButton
```

Comment:
This logic is duplicated in `AutoEntityButton` and `EntitySelectButton`, and it could make sense to implement it in a dedicated component. But as it will probably change very soon with the new mobile UI, I suppose that's not important.

Reply:
It's possible that this part of the code will not change a lot. I'm going to create an issue to not forget to clean the duplicated code after the refactor.